src/quality-level-list.js

```diff
@@ -103,7 +106,7 @@ class QualityLevelList extends videojs.EventTarget {
   /**
    * Removes a quality level from the list.
    *
-   * @param {QualityLevel} remove QualityLevel to remove to the list.
+   * @param {QualityLevel} qualityLevel QualityLevel to remove to the list.
```

Comment:
```suggestion
   * @param {QualityLevel} qualityLevel  The QualityLevel to remove from the list.
```

Reply:
Thanks @mister-ben. I have just updated the PR.